src/PatientChecking/PatientChecking/Services/ImageService.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Newtonsoft.Json;
+using PatientChecking.Services.Repository;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace PatientChecking.Services
+{
+    public class ImageService : IImageService
+    {
+        private readonly IHostingEnvironment _hostingEnvironment;
+
+        public ImageService(IHostingEnvironment hostingEnvironment)
+        {
+            _hostingEnvironment = hostingEnvironment;
+        }
+
+        public bool IsImageFile(IFormFile formFile)
+        {
+            var fileExtension = Path.GetExtension(formFile.FileName).ToLower();
+
+            if (fileExtension != ".jpg" && fileExtension != ".png" && fileExtension != ".jpeg")
+            {
+                return false;
+            }
+
+            return true;
+        }
+
+        public string SaveImage(IFormFile formFile)
+        {
+            string uploadsFolder = Path.Combine(_hostingEnvironment.WebRootPath, "Image");
```

Comment:
What happens if image is blocked or can't be copied?
Should handle this logic under `try {} catch {}`

Reply:
Code has been handled under try catch. Please review this @vietvo361